packages/frontend/src/redux/slices/account/createAccountThunks.js

```diff
@@ -219,3 +228,70 @@ export const finishLocalSetupForZeroBalanceAccount = createAsyncThunk(
 
     }
 );
+
+export const initiateSetupForZeroBalanceAccountPhrase = createAsyncThunk(
+    `${SLICE_NAME}/initiateSetupForZeroBalanceAccountPhrase`,
+    async ({
+        implicitAccountId,
+        recoveryKeyPair
+    }, { dispatch }) => {
+        try {
+            try {
+                await sendJson('POST', `${ACCOUNT_HELPER_URL}/account/seedPhraseAdded`, {
+                    accountId: implicitAccountId,
+                    publicKey: recoveryKeyPair.publicKey.toString()
+                });
+            } catch (e) {
+                if (e.status === 400) {
```

Comment:
```suggestion
                if (e.message.includes('ConditionalCheckFailedException')) {
```
contract-helper throws `400` by default so it may be better to check the message explicitly. Ideally contract-helper would throw a specific error relating to creating duplicates but best we can do now is check for the DynamoDB error.

Thoughts @andy-haynes ?

Reply:
Yep I'll use `ConditionalCheckFailedException` for now and we can update if we end up adding better error in contract-helper later